src/test/java/se/liu/ida/hefquin/engine/wrappers/lpgwrapper/SPARQLStar2CypherTranslatorTest.java

```diff
@@ -0,0 +1,333 @@
+import se.liu.ida.hefquin.engine.wrappers.lpgwrapper.impl.DefaultConfiguration;
+import se.liu.ida.hefquin.engine.wrappers.lpgwrapper.query.CypherQuery;
+import se.liu.ida.hefquin.engine.wrappers.lpgwrapper.query.CypherVar;
+import se.liu.ida.hefquin.engine.wrappers.lpgwrapper.query.impl.CypherUnionQueryImpl;
+import se.liu.ida.hefquin.engine.wrappers.lpgwrapper.query.impl.condition.*;
+import se.liu.ida.hefquin.engine.wrappers.lpgwrapper.query.impl.match.EdgeMatchClause;
+import se.liu.ida.hefquin.engine.wrappers.lpgwrapper.query.impl.match.NodeMatchClause;
+import se.liu.ida.hefquin.engine.wrappers.lpgwrapper.query.impl.returns.*;
+import se.liu.ida.hefquin.engine.wrappers.lpgwrapper.utils.CypherQueryBuilder;
+import se.liu.ida.hefquin.engine.wrappers.lpgwrapper.utils.SPARQLStar2CypherTranslator;
+
+import static org.junit.Assert.assertEquals;
+
+public class SPARQLStar2CypherTranslatorTest {
+
+    final CypherVar v1 = new CypherVar("cpvar1");
+    final CypherVar v2 = new CypherVar("cpvar2");
+
+    final CypherVar ret1 = new CypherVar("ret1");
+    final CypherVar ret2 = new CypherVar("ret2");
+    final CypherVar ret3 = new CypherVar("ret3");
+
+    final CypherVar src1 = new CypherVar("src1");
+    final CypherVar edge1 = new CypherVar("edge1");
+    final CypherVar tgt1 = new CypherVar("tgt1");
+
+    final CypherVar a1 = new CypherVar("a1");
+    final CypherVar a2 = new CypherVar("a2");
+    final CypherVar a3 = new CypherVar("a3");
+    final CypherVar a4 = new CypherVar("a4");
+    final CypherVar a5 = new CypherVar("a5");
+    final CypherVar a6 = new CypherVar("a6");
+    final CypherVar a7 = new CypherVar("a7");
+    final CypherVar a8 = new CypherVar("a8");
+
+    final LPGNode node22 = new LPGNode("22", null, null);
+    final LPGNode node23 = new LPGNode("23", null, null);
+
+    @Test
+    public void translateVarPropertyLiteralTest() {
+        final LPG2RDFConfiguration conf = new DefaultConfiguration();
+        final Triple tp = new Triple(Var.alloc("s"), conf.mapProperty("name"),
+                NodeFactory.createLiteral("Quentin Tarantino"));
+        final CypherQuery translation = SPARQLStar2CypherTranslator.translateTriplePattern(new TriplePatternImpl(tp), conf);
+        assertEquals(
+                new CypherUnionQueryImpl(
+                    CypherQueryBuilder.newBuilder().match(new NodeMatchClause(v1))
+                            .condition(new PropertyValueCondition(v1, "name", "Quentin Tarantino"))
+                            .returns(new VariableReturnStatement(v1, ret1))
+                            .build(),
+                    CypherQueryBuilder.newBuilder()
+                            .match(new EdgeMatchClause(src1, edge1, tgt1))
+                            .condition(new PropertyValueCondition(edge1, "name", "Quentin Tarantino"))
+                            .returns(new TripleMapReturnStatement(src1, edge1, tgt1, ret1))
+                            .build()), translation);
```

Comment:
Please move `translation` to a new line, at the same indentation level as the `new` on line 54. That way, it will be much easier to see what is supposed to be equal to what. (Same thing for the other unit tests below.)